app/screens/LoginScreen.js

```diff
@@ -77,11 +77,11 @@ const styles = StyleSheet.create({
     fontSize: fonts.sizeXL,
     fontWeight: 'bold',
     paddingVertical: 50,
-    fontFamily: Platform.OS === 'ios' ? 'AvenirNext-Heavy' : 'Roboto'
+    fontFamily: Platform.OS === 'ios' ? fonts.titleFamilyIos : fonts.titleFamilyAndroid
   },
   inputContainer: {
-    width: '100%',
-    padding: 15,
+    alignSelf: 'stretch',
+    padding: Platform.OS === 'ios' ? 15 : 0,
     marginVertical: 10,
     backgroundColor: colors.grayLight,
     borderRadius: 15,
```

Comment:
I noticed there is no `KeyboardAvoidingView` being used in this screen.  This will probably be needed when using a real device.  By default, the view does not change when the keyboard comes into the screen, so the buttons may be covered by the keyboard.  You may also need to implement a handler to close the keyboard if a touch is detected outside the input field.  You can test this by enabling the `Software Keyboard` in the iOS simulator. "I/O -> Keyboard -> Enable Software Keyboard"